frameworks/Java/servicetalk/pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.techempower</groupId>
+    <artifactId>servicetalk</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>io.servicetalk</groupId>
+            <artifactId>servicetalk-serialization-api</artifactId>
+            <version>0.20.0</version>
```

Comment:
Usually when we have a bunch of related dependencies that are supposed to all have the same version like this, we extract that version number into a property. Please do that here.

```xml
<properties>
  <servicetalk.version>0.20.0</servicetalk.version>
</properties>

<dependencies>
  <dependency>
    <groupId>io.servicetalk</groupId>
    <artifactId>servicetalk-http-api</artifactId>
    <version>${servicetalk.version}</version>
  </dependency>
  ...
</dependencies>
```

Often times projects will extract all dependency version numbers into properties even if each of those version numbers is only referenced once in the pom file.